templates/home.html

```diff
@@ -70,6 +70,13 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
           <span class="ml small">ദുരിതാശ്വാസ ക്യാമ്പുകൾ</span>
         </span>
       </a>
+    <a href="/announcements" class="home-button card" role="button">
+        {% bootstrap_icon "bullhorn" %}
+        <span class="text">
+        Announcements<br/>
+            <span class="ml small">പ്രഖ്യാപനം</span>
```

Comment:
I think "അറിയിപ്പുകൾ " would be better Malayalam equivalent

Reply:
Sure updating it.